devicetypes/smartthings/qubino-meter.src/qubino-meter.groovy

```diff
@@ -0,0 +1,262 @@
+/**
+ *  Copyright 2015 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Qubino Meter
+ *
+ *  Author: WarrenK-design https://github.com/WarrenK-design
+ *
+ *  Date: 06/10/2020
+ */
+metadata {
+	definition (name: "Qubino Meter", namespace: "smartthings", author: "WarrenK-design", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, ocfDeviceType: "x.com.st.d.energymeter") {
```

Comment:
Since this Qubino device and other Aeon devices are already supported in DTH: "Aeon Home Energy Meter"  - if there are any changes (regarding Your device) that must be added to the code, they should be added in:

https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/aeon-home-energy-meter.src/aeon-home-energy-meter.groovy

Please keep in mind that "tiles" are not supported in SmartThings app anymore (they were supported in SmartThings Classic only).

Reply:
Hi @KKlimczukS,

Sorry I'm new to SmartThings and was just curious of the process. There are a few measurement readings that the device handler doesn't support however and that I have implemented in my own local pull of the SmartThings public repo. Is the [main repository](https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/aeon-home-energy-meter.src/aeon-home-energy-meter.groovy) open to anyone to submit changes or would I have to work at Qubino, Aeon etc.

Reply:
Hi @WarrenK-design, 

The DTH handles only those measurement readings that are used by devices supported by this DTH (aeon-home-energy-meter.groovy). When it comes to  Aeon and Qubino energy monitors - [kWh], [kVAh] and [W] are sufficient here.

Reply:
Currently, new integrations are driven by WWST certification program ( DTHs are developed internally) or CST certification program (changes in DTH are made by manufacturers):
https://smartthings.developer.samsung.com/docs/devices/publishing/publishing-basics.html